.circleci/config.yml

```diff
@@ -106,3 +108,17 @@ jobs:
             docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_SHA1
             docker push quay.io/plotly/image-exporter:$CIRCLE_BRANCH
+
+  electron-pack-and-release:
+    docker:
+      - image: circleci/node:8-browsers
+    steps:
+      - checkout
+      - run:
+          name: Install deps
+          command: yarn install
+      - run:
+          name: Electron-Builder pack
+          command: yarn run pack
+      - store_artifacts:
+          path: release
```

Comment:
We don't really want to include `release/linux-unpacked` (131MB).
See how Falcon does it:
https://github.com/plotly/falcon-sql-client/blob/3ebfd4d20a14973f6023990fabfff0cc285e0b21/circle.yml#L37-L41

Reply:
Okay @n-riesco, I believe you'll like it now: https://circleci.com/gh/plotly/image-exporter/1172#artifacts/containers/0. What do you think?